src/wp-content/themes/twentyfifteen/css/editor-blocks.css

```diff
@@ -214,7 +224,12 @@ Description: Used to style blocks in the editor.
 		padding: 0.65em 0 0.3em;
 	}
 
-	.edit-post-visual-editor .editor-block-list__block h5,
+	.edit-post-visual-editor .editor-block-list__block h5 {
+		font-size: 16px;
+		line-height: 1.25;
+		padding: 0.8125em 1.625em;
```

Comment:
Could you update this to have zero padding on the left and right sides at the `59.6875em` breakpoint?
`padding: 0.8125em 0 1.625em;`

Reply:
@sabernhardt Thanks for your feedback, I updated it in this commit. https://github.com/WordPress/wordpress-develop/pull/2496/commits/39073e279739dae30394bc2b6a5ffc1b8c3a13aa

Reply:
OK, thanks for your help. Do you mind if I rebase this commit https://github.com/WordPress/wordpress-develop/commit/39073e279739dae30394bc2b6a5ffc1b8c3a13aa to adjust the padding?

Reply:
I would add another commit, though I don't know whether that is better than rebasing.

Reply:
I'd prefer to use `padding: 0.875em 0 0.4375em;` , thanks!